### DISCUSSION TITLE
Wall thickness

### DISCUSSION THREAD

**Discussion by ljungis** [supermerill/SuperSlicer#3599] [Q&A]
Hi, I'm new to Superslicer, coming from Ideamaker. 
The part will be printed with 0.6 mm nozzle. The walls are 1,2 mm thick.
Looking at the attached screenshot it looks like the wall in the center of the part only have 1 perimeter and the others 2.
After printing when I measure the thickness of that wall, it is thinner. What setting do I have to change to get it correct?
Thankful for help.

![Skärmbild från 2023-01-24 13-22-57](https://user-images.githubusercontent.com/37312627/214290908-4b31cfd7-bc74-4bfc-886a-db68e090e772.png)
[Part1.STL.zip](https://github.com/supermerill/SuperSlicer/files/10489499/Part1.STL.zip)
[PETG.ini.zip](https://github.com/supermerill/SuperSlicer/files/10489667/PETG.ini.zip)

**Comment by neophyl** [comment]
Without you attaching your saved and zipped up project file its hard to know.  A 'normal' profile generally has a slightly wider extrusion width defined than the nozzle, so for example a 0.4mm nozzle will generally have 0.45mm extrusion width.  Who knows what you have configured.  The petg.ini is pointless and the stl isnt mush more use without having a copy of all your settings.  And the best way to do that is to attach a project file.

Also what version of SuSi are you using ?  2.4.x or 2.5.xx as 2.5 versions have arachne available which handles extrusion widths differently.

Basically save your project from SuSi (File>Save project As) and then zip up the resulting 3mf file and attach it.  Has to be zipped or the file type isnt allowed.  A project is a complete snap shot and so allows anyone to slice with exactly the same settings you are using.

**Comment by ljungis** [comment]
Thanks

My Version is 2.4.58.5
[Part1.3mf_FILES.zip](https://github.com/supermerill/SuperSlicer/files/10491692/Part1.3mf_FILES.zip)

**Comment by neophyl** [comment]
When I unzip the archive there isnt a 3mf I can import.  If I rename the zip to be a .3mf and open it, it doesn't load a part or a profile in.  So unfortunately I cant use the attached.

You have to create a project in Super Slicer with the issues you are observing,  make sure its slicing etc and showing the problems you describe then use File>Save as to save a .3mf file.  Then zip that up.

**Comment by ljungis** [comment]
Hope this will work!

[Part1.zip](https://github.com/supermerill/SuperSlicer/files/10492277/Part1.zip)

**Comment by neophyl** [comment]
I can at least open that one.  Unfortunately when I slice it I am getting an access violation error which shuts SuSi down.  I'll try and figure out what setting its choking on.

**Comment by ljungis** [comment]
Hi , here is another .3mf file you can try.

[Part1.zip](https://github.com/supermerill/SuperSlicer/files/10497500/Part1.zip)
